src/cli/existing_mnemonic.rs

```diff
@@ -32,9 +32,9 @@ pub fn subcommand<'a, 'b>() -> App<'a, 'b> {
         .arg(
             Arg::with_name("chain")
                 .long("chain")
-                .required(true)
+                .required(false)
```

Comment:
why isn't it required anymore?

Reply:
Because it's possible to pass `--testnet_config` now instead of `--chain`